store/storetest/channel_store.go

```diff
@@ -4610,8 +4610,11 @@ func testGetMemberCountsByGroup(t *testing.T, ss store.Store) {
 		} else if i == 3 || i == 4 {
 			timeZone["manualTimezone"] = "PST"
 			timeZone["useAutomaticTimezone"] = "false"
-		} else if i == 5 || i == 6 {
-			timeZone["autoTimezone"] = "PST"
+		} else if i == 5 {
+			timeZone["autoTimezone"] = "CET"
```

Comment:
To clarify: we are deliberately setting `autoTimezone` instead of `automaticTimezone` to test for failure?

Reply:
Yes. I am thinking that was the intent in the previous test so I kept it.